documentation/PublicAPI.md

```diff
@@ -0,0 +1,85 @@
+PUBLIC API Specification
+--------------------------
+
+The following API endpoints are designed to be publicly available and used by third parties as well as by the frontend itself. The data format used by all the endpoints is JSON.
+
+## GET /api/tools
+
+Returns a JSON document with descriptions of all tools registered with the Switchboard. Each tool has a unique (for the Switchboard) numeric ID, which can be used to identify the tool.
+
+## GET /api/tools/{id}
+
+Returns a JSON document with the description of the tool registered with the Switchboard that has the particular **id** used as a path parameter.
+
+Example:
+```
+GET https://switchboard.clarin.eu/api/tools/112
+
+Returns:
+{
+    "formatVersion":"2",
+    "id":112,
+    "task":"Text Analytics",
+    "name":"WebLicht Advanced Mode",
+    ...
+}
+```
+
+## POST /api/tools/match
```

Comment:
Something could be said in this one also about paging and filtering

Reply:
added few words about that